### PR TITLE
Allow slack bots to be admins and participate in games

### PR DESCRIPTION
**Pull request by phelios**
Mostly refactoring changes, but the main feature change is the ability to detect comments from bots as player and convert their default bot_id to user_id version.

PS: The actual BDD is in another repository. Still very basic, able issue commands but don't understand responses yet.

### CODE REVIEW

File: emojirades/tests/FileFixture.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    """
+    Test helper to read fixtures files with relative path
+    path: pass in the path after `fixtures`
+    """
+    def __init__(self, path):
+        dirname = os.path.dirname(__file__)
+        self.filename = os.path.join(dirname, os.path.join("fixtures", path))
+
+    def open(self):
+        return open(self.filename, 'r')
```
Comment (michael-robbins) [inline emojirades/tests/FileFixture.py:14]: I'm a bit of a fan of double quotes, black run maybe?

**Review by michael-robbins — APPROVED** [review]
Fine with it structurally, I think just a black run over it and we're sweet

File: emojirades/tests/FileFixture.py (hunk 1 above)
Comment (phelios) [reply emojirades/tests/FileFixture.py:14]: argh.. old habit lol.. oh we have 'black', don't we? haha.. will do that before squashing.

**Comment by phelios** [comment]
Not sure if I like Black anymore, seem to find more things to fix every time. And it's incompatible with pycodestyle 😞 .

**Review by michael-robbins — APPROVED** [review]

**Pull request merged**